WindowManager/app/src/main/java/com/example/windowmanagersample/SplitLayoutActivity.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2021 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.windowmanagersample;
+
+import android.os.Bundle;
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.core.util.Consumer;
+import androidx.window.WindowInfoRepo;
+import androidx.window.WindowLayoutInfo;
+import androidx.window.java.WindowInfoRepoJavaAdapter;
+import com.example.windowmanagersample.databinding.ActivitySplitLayoutBinding;
+
+public class SplitLayoutActivity extends AppCompatActivity {
```

Comment:
Is there a reason we swapped this back to java for the sample? Just to showcase the JavaAdapter?

Reply:
Yes, the idea was to showcase how to use the JavaAdapter's API.